types.ts

```diff
@@ -0,0 +1,15 @@
+export interface TodoCardInfoStructure {
```

Comment:
you can remove Structure word from naming, TodoCardInfo and TodoFormInput is enough